File: Python PowerPoint Examples/NotesSlidesApiDemo/getNotesSlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.notes_slides_api.NotesSlidesApi(configuration)

name = "GetNotesSlide.pptx"
password = ""
storage = ""
folder = "input"
slideIndex = 0

result = api.get_notes_slide(name=name, slide_index=slideIndex,
                             password=password, storage=storage,
                             folder=folder)

File: PHP PowerPoint Examples/NotesSlidesApiDemo/getNotesSlide.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\NotesSlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new NotesSlidesApi($configuration);

$name = "GetNotesSlide.pptx";
$password = null;
$folder = "Powerpoint";
$storage = null;
$slideIndex = 0;
$result = $apiInstance->getNotesSlide($name,$slideIndex, $password, $folder, $storage);
print_r($result);